graphql/resolvers.ts

```diff
@@ -85,6 +85,7 @@ const resolvers = {
         }
         return data;
       } catch (error) {
+        console.log(error)
```

Comment:
you should add a context in the log.
ie: `console.error({}, "error in abc place")` so that is more understand where the error come from when looking at the logs